examples/1_simple_runbooks/backup.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+from brigade.easy import easy_brigade
```

Comment:
mostly defaulting to the safest options which are (1) just give me the changes without changing anything and (2) if something bad happens break! I am happy to discuss alternatives for each option but the reasoning was just simply that "default to the safest scenario if the user ignores these parameters"

Reply:
I think it is more logical to default to the most common/wanted option, not the safest. 

Also there will be plenty of tasks that won't have a dry-run so defaulting to dry-run doesn't make sense as a default.

I think this is the behavior most people would expect also.

Reply:
re `dry_run`; sounds fine by me. We can change the default to `False`. What's your take on `raise_on_error`?

Reply:
I agree regarding `dry_run`. For `raise_on_error` I don't think it's as important as it was as a host should be marked as skipped for the following tasks it shouldn't be as destructive by default as it could have been previously.